patch/kernel/rockchip-rk3588-edge/0042-arm64-dts-rockchip-rk3588-Add-GPU-nodes.patch

```diff
@@ -0,0 +1,150 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Boris Brezillon <boris.brezillon@collabora.com>
+Date: Mon, 7 Aug 2023 17:30:58 +0200
+Subject: arm64: dts: rockchip: rk3588: Add GPU nodes
+
+Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
+---
+ arch/arm64/boot/dts/rockchip/rk3588s.dtsi | 119 ++++++++++
+ 1 file changed, 119 insertions(+)
+
+diff --git a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
+index 762a095648b1..f43f10340d5d 100644
+--- a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
++++ b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
+@@ -962,6 +962,120 @@ usb_host2_xhci: usb@fcd00000 {
+ 		snps,dis-del-phy-power-chg-quirk;
+ 		snps,dis-tx-ipgap-linecheck-quirk;
+ 		snps,dis_rxdet_inp3_quirk;
```

Comment:
Here remove the "disabled" status 
of usb_host2_xhci, this may cause a boot issue on some boards that use combphy2_psu  phy for other functions.

Reply:
It does not remove the disabled line, just the patch looks like it has removed that line.

Reply:
After rechecking it this patch does have issue. I will update later.